crates/matrix-sdk-crypto/src/machine/mod.rs

```diff
@@ -1409,7 +1434,7 @@ impl OlmMachine {
         e: ToDeviceEvent<ToDeviceEncryptedEventContent>,
     ) -> Option<ProcessedToDeviceEvent> {
         let decrypted = match self.decrypt_to_device_event(transaction, &e, changes).await {
-            Ok(e) => e,
+            Ok(decrypted) => decrypted,
```

Comment:
To remove the `?` after the `match`, you could do this instead:

```suggestion
            Ok(maybe_decrypted) => maybe_decrypted?,
```

Reply:
Yeah, I much prefer that too. Thanks! eb287c1750